doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NAME | String | Yes | Server name should be aligned with naming rule, this is the unique ID for each device to be referred for identify device |
+| VENDOR | String | Yes | Vendor of device |
+| SKU | String | Yes | The SKU of device, can be referred by other table NIC connection table, to identify slot-port mapping for device  |
+| MODEL | String | Yes | This is the model for different service type, this value is defined from design document of resource pool, example NC1, NC1-S |
+| SN | String | Yes | Serial Number |
+| RES_POOL | String | Yes | Resource pool name |
+| RACK | String | Yes | rack name where device located |
+| POS | String | Yes | the position of device in rack, like 2-3U,4-5U |
+| BMC_IP | String | Yes |  |
+| BMC_GATEWAY | String | Yes |  |
+| BMC_MASK | String | Yes |  |
+| BMC_SUBNET | String | Yes |  |
+| BMC_USR | String | Yes | BMC user |
+| BMC_PWD | String | Yes | BMC password |
+| SERVICE_IP | String | Yes |  |
+| SERVICE_GATEWAY | String | Yes |  |
+| SERVICE_MASK | String | Yes |  |
+| GROUP_NAME | String | Yes | the usage of server, Manage or Storaage or Service  |
+| BMC_PRE_CONFIGURED | String | Yes | YES or NO |
+| HW_REGION | String | Yes | hardware region that divided by design documents, like A area or B area |
+| MODULE_NAME | String | Yes | hardware model that divided within each region, Like "Model 3 in Region A", usually contains certain number of racks |
+
+<p align="center"><b>Table 6-3-4-1:</b> Server Information.</p>
+
+#### 6.3.4.2 server nic information
+This table is describing the slot and port mapping relationship for NIC in each model of server. 
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes | Vendor of server |
+| SKU | String | Yes | SKU of server |
+| MODEL | String | Yes | server service type defined by provider/user, same definition as in above table, example: NC1 or NC2  |
+| SLOT | String | Yes | Slot number in server for each NIC, for example, PCIeSlot2  |
+| PORTS | String | Yes | Ports number for the above NIC, for example: 1_1;1_2 2 ports for one NIC  |
+| SLOT | String | Yes | Slot number in server for each NIC, for example, PCIeSlot2  |
+
+<p align="center"><b>Table 6-3-4-2:</b> Server NIC Information.</p>
+
+#### 6.3.4.3 Port BDF information for each type pf server
+Port BDF information need to be provided for each port on server, 
+it will be used to identify the logical port name after OS is installed. 
```

Comment:
I will add the slot ID, like PCIe1/2, it was included in the initial definition, but somehow it was missed after some version updates..
We thought about using MAC address, MAC will be unique for each port, so it would be a lot of data. 
So then we decided to provide BDF from each type of server in PDF,  then get MAC address and corresponding logical name when installer need to get logical name.